food.css

```diff
@@ -0,0 +1,20 @@
+h1,
+h2,
+h4 {
+	text-align: center;
+}
+h1 {
+	margin-top: 30px;
+}
+
+h2 {
+	margin-top: 0;
+}
+
+.innerRow {
+	border: .5px solid black;
+	padding-bottom: 15px;
+}
+.innerRows{
+	border: .3px solid darkgrey;
+}
```

Comment:
Spacing is a little inconsistent on this file, with blank lines between some selector sets and not others.